docs/source/platform/schema-registry.md

```diff
@@ -77,24 +75,26 @@ To get the most out of using tagged schemas, teams can send metrics to [Engine](
 const server = new ApolloServer({
   // rest of normal server settings
   engine: {
-    schemaTag: 'beta',
-  },
+    schemaTag: "beta"
+  }
 });
 ```
 
-Both the new version of the schema, as well as its performance and error metrics can be viewed using [Engine](https://engine.apollographql.com) and can even be used with [schema validation](./schema-validation.html).
+Both the new version of the schema, as well as its performance and error metrics can be viewed in the [Apollo UI](https://engine.apollographql.com) and used with [schema validation](./schema-validation.html).
 
 <h2 id="history">Schema history</h2>
 
-As your schema grows and evolves to meet the needs of your product, it is helpful to see a history of changes for a team. This allows everyone to know when new features were introduced, when old fields were removed, and even link back to the commit that caused the change. Apollo Engine provides all the tooling needed to track this history in a simple way. Every time your schema is updated, you can simply run the [`apollo service:push`](#publish) command to keep an up to date history of your schema.
+As the schema grows and evolves to meet the needs of the product, keeping and visualizing the history of schema changes becomes increasingly valuable. A consistent historical view allows everyone to understand when new features were introduced, when old fields were removed, and which commit made a change. The Apollo Platform provides the tooling necessary to track this history with the [`apollo service:push`](#publish) command.
+
+After a schema is published, it becomes the basis for comparison when validating future schemas to avoid breaking changes. Therefore, a service should be pushed to [Engine](https://engine.apollographql.com) each time a new schema is deployed.
```

Comment:
published? push? I think it'd be better if we can stick to one word (now that we're here! Again, I know that these are just great copy changes you're making so far, but why not make them even better? 😉 )

Reply:
push a service and tag. Register a schema